thoth/metrics_exporter/jobs.py

```diff
@@ -37,275 +37,394 @@
+        )
+        metrics.graphdb_total_python_packages_with_solver_error.set(
+            total_python_packages_with_solver_error_unparsable + total_python_packages_with_solver_error_unsolvable
+        )
+
+        _LOGGER.debug(
+            "graphdb_total_python_packages_with_solver_error=%r",
+            total_python_packages_with_solver_error_unparsable + total_python_packages_with_solver_error_unsolvable,
+        )
+
+        _LOGGER.debug(
+            "graphdb_total_python_packages_with_solver_error_unparsable=%r",
+            total_python_packages_with_solver_error_unparsable,
+        )
+
+        _LOGGER.debug(
+            "graphdb_total_python_packages_with_solver_error_unsolvable=%r",
+            total_python_packages_with_solver_error_unsolvable,
+        )
+
+
+class InspectionMetrics:
+    """Class to evaluate Metrics for Amun Inspections."""
+
+    def get_inspection_results_per_identifier(self):
+        """Get the total number of inspections in Ceph per identifier."""
+        store = InspectionResultsStore()
+        if not store.is_connected():
+            store.connect()
+
+        specific_list_ids = {}
+        specific_list_ids["without_identifier"] = 0
+        for ids in store.get_document_listing():
+            inspection_filter = "_".join(ids.split("-")[1:(len(ids.split("-")) - 1)])
+            if inspection_filter:
+                if inspection_filter not in specific_list_ids.keys():
+                    specific_list_ids[inspection_filter] = 1
+                else:
+                    specific_list_ids[inspection_filter] += 1
+            else:
+                specific_list_ids["without_identifier"] += 1
+
+        for identifier, identifier_list in specific_list_ids.items():
+            metrics.inspection_results_ceph.labels(identifier).set(identifier_list)
+            _LOGGER.debug(f"inspection_results_ceph for {identifier} ={identifier_list}")
+
+    def get_inspection_python_software_stack_count(self):
+        """Get the total number of Inspection Python Software Stacks in Thoth Knowledge Graph."""
+        graph_db = GraphDatabase()
+        graph_db.connect()
+
+        thoth_graphdb_total_inspection_software_stacks = graph_db.python_software_stack_count(
+            software_stack_type="INSPECTION"
+        )
+        metrics.graphdb_inspection_software_stacks_records.set(thoth_graphdb_total_inspection_software_stacks)
+        _LOGGER.debug("graphdb_inspection_software_stacks_records=%r", thoth_graphdb_total_inspection_software_stacks)
+
+
+class AdviserMetrics:
+    """Class to evaluate Metrics for Adviser."""
+
+    def get_advised_python_software_stack_count(self):
+        """Get the total number of Advised Python Software Stacks in Thoth Knowledge Graph."""
+        graph_db = GraphDatabase()
+        graph_db.connect()
+
+        thoth_graphdb_total_advised_software_stacks = graph_db.python_software_stack_count(
+            software_stack_type="ADVISED"
```

Comment:
From method/function design PoV, I would recommend creating an enum for this in thoth-storages. It will be espiecially useful when we do type checking to verify we query for existing enums in the database.